classes/itembase.php

```diff
@@ -142,8 +142,8 @@ class itembase {
         'variable' => true,
         'indent' => true,
         'hidden' => true,
-        'reserved' => true,
```

Comment:
This was added because of an error (because I can never do one job per time). This line shouldn't be here and I use this branch to drop it.